l10n_ch_qr_bill_without_amount/views/payment_mode.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_payment_mode_form" model="ir.ui.view">
+        <field name="name">account.payment.mode.form</field>
+        <field name="model">account.payment.mode</field>
+        <field name="inherit_id" ref="account_payment_mode.account_payment_mode_form" />
```

Comment:
account_payment_mode module isn't in the manifest file

Reply:
dependency account_payment_mode already in account_payment_partner